lib/grammers-mtproto/src/transport/abridged.rs

```diff
@@ -99,11 +99,11 @@ impl Transport for Abridged {
         let header_len = header_len as usize;
         let len = len as usize;
 
-        Ok(UnpackedOffset {
+        Ok(vec![UnpackedOffset {
```

Comment:
I'm not sure forcing an allocation like this, as cheap as it might be in practice, is worth it.